### ISSUE TITLE
Title (V3) missing linkAccessibilityLabel and linkTitleAttribute

### ISSUE THREAD

**Issue by yudei-rojas** [adobe/aem-core-wcm-components#2841]
## Bug Report

**Current Behavior**
Title (V3) readme.md mentions that linkAccessibilityLabel  and linkTitleAttribute are "Edit Dialog Properties" and they are not.

**Expected behavior/code**
When migrating from title V2 to V3 I'm losing  linkAccessibilityLabel  and linkTitleAttribute and won't be able to add/edit them anymore.

**Environment**
- AEM version: 6.5.19
- Core Components version: 2.23.4
- JRE version : openjdk version "1.8.0_422"

**Possible Solution**
It looks this below was a solution, however it didn't make it to any release?
https://github.com/adobe/aem-core-wcm-components/pull/1538/files

**Additional context / Screenshots**
Title (V2):

![image](https://github.com/user-attachments/assets/11ffbae1-19b2-45c0-9042-faf00b51d157)

Title(V3)

![image](https://github.com/user-attachments/assets/3c161664-d3a9-4d49-b10f-1e1ab1078c3d)
